packages/react/src/components/InlineAlert/index.stories.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { withKnobs, select, text } from "@storybook/addon-knobs";
+import { action } from "@storybook/addon-actions";
+import { withInfo } from "@storybook/addon-info";
+
+import InlineAlert from "./";
+import { INLINE_ALERT_TYPES } from "../../constants/";
+
+storiesOf("Alerts")
```

Comment:
According to the Notion, there should be:
- A story with all states
- A `Playground` story

Reply:
Absolutely!